nessie/src/main/java/org/apache/iceberg/nessie/NessieIcebergClient.java

```diff
@@ -530,6 +599,17 @@ private String buildCommitMsg(TableMetadata base, TableMetadata metadata, String
     return String.format("Iceberg commit against %s", tableName);
   }
 
+  private String buildCommitMsgForView(ViewMetadata base, ViewMetadata metadata, String viewName) {
+    String operation = metadata.currentVersion().operation();
+    if (base != null && !metadata.currentSchemaId().equals(base.currentSchemaId())) {
+      return String.format(
+          "Iceberg schema change against %s for the operation %s", viewName, operation);
+    } else if (base == null) {
+      return String.format("Iceberg view created with name %s", viewName);
+    }
+    return String.format("Iceberg commit against %s for the operation %s", viewName, operation);
```

Comment:
rather than having two cases for a create/replace, you could just use the `operation` from above: `return String.format("Iceberg view %sd with name %s", operation, viewName);`